woocommerce/google-listings-and-ads
# Fix the giant icon problem in the warning messages

### Changes proposed in this Pull Request:



This problem was discovered in #527.

![image](https://user-images.githubusercontent.com/17420811/116394736-df7b0500-a855-11eb-9f91-463bdd04945f.png)


### Root cause
Libraries depend on a different version of `@wordpress/icons`, causing the react component instance checking `icon.type === SVG` to get false result.

https://github.com/WordPress/gutenberg/blob/%40wordpress/components%4012.0.8/packages/components/src/icon/index.js#L41-L50

### Solution

Upgrade `@wordpress/icons` to the same version used in `@wordpress/components`.

### Screenshots:

![image](https://user-images.githubusercontent.com/17420811/116395236-721ba400-a856-11eb-8aea-95ac7668370a.png)


### Detailed test instructions:

1. `npm install` and restart dev build `npm start`
2. Set the store location to “South Sudan” in WooCommerce → Settings
3. Head to the Get Started page and the external link icon should be displayed with a proper size.
